containers/Singularity/rocky-8.def

```diff
@@ -14,4 +14,11 @@ Include: dnf
     shadow-utils rsyslog tzdata util-linux words zlib tar less \
     gzip which util-linux openssh-clients openssh-server dhclient \
     pciutils vim-minimal shadow-utils strace cronie crontabs cpio \
-    wget rocky-release ipmitool yum NetworkManager
\ No newline at end of file
+    wget rocky-release ipmitool yum NetworkManager network-scripts
```

Comment:
Can we just leave off NetworkManager?  If so we can skip the disables, below too.

Reply:
Yep, good thinking. I'll do that.

Thanks!

Reply:
Had some issues this morning with inconsistent eth0/eth1 device naming using this setup.  See warewulf slack thread.. I ended up adding back NetworkManager and excluding all devices except warewulf manged ones.. and still managed to have issues until I changed names from eth0 to ww0.   So not sure if I would've ended up in the same place using network-scripts.  Just not sure it's the way to go.